src/navigation/components/NavigationItem/components/ContentWrapper/ContentWrapper.tsx

```diff
@@ -0,0 +1,16 @@
+import React, {Fragment} from 'react';
+
+import {block} from '../../../../../utils';
+import {Image} from '../../../../../components';
+import {ImageProps} from '../../../../../models';
+
+import './ContentWrapper.scss';
+
+const b = block('content-wrapper');
+
+export const ContentWrapper: React.FC<{text: string; icon?: ImageProps}> = ({text, icon}) => (
+    <Fragment>
+        {icon && typeof icon !== 'string' && <Image className={b('icon')} {...icon} />}
```

Comment:
We have a component Icon with size prop. Can we use it here?

Reply:
I think not, because `Icon` waiting another props. I think if we wants use in this place `Icon` we needs another PR to do it.